docs/api-docs/storefront/graphql/graphql-carts-and-checkout.mdx

```diff
@@ -0,0 +1,1073 @@
+            "billingAddress": null,
+            "shippingConsignments": [],
+            "order": null,
+            "shippingCostTotal": {
+              "value": 0,
+              "currencyCode": "USD"
+            },
+            "giftWrappingCostTotal": {
+              "value": 0,
+              "currencyCode": "USD"
+            },
+            "handlingCostTotal": {
+              "value": 0,
+              "currencyCode": "USD"
+            },
+            "taxTotal": {
+              "value": 0,
+              "currencyCode": "USD"
+            },
+            "taxes": [
+              {
+                "name": "Tax",
+                "amount": {
+                  "value": 0,
+                  "currencyCode": "USD"
+                }
+              }
+            ],
+            "subtotal": {
+              "value": 59.95,
+              "currencyCode": "USD"
+            },
+            "grandTotal": {
+              "value": 59.95,
+              "currencyCode": "USD"
+            },
+            "createdAt": {
+              "utc": "2023-06-01T17:45:23Z"
+            },
+            "updatedAt": {
+              "utc": "2023-06-01T19:07:15Z"
+            },
+            "customerMessage": "",
+            "outstandingBalance": {
+              "value": 59.95,
+              "currencyCode": "USD"
+            },
+            "coupons": [],
+            "promotions": []
+          }
+        }
+      }
+    }
+    ```
+  </Tab>
+</Tabs>
+
+### Add checkout billing address
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    ```graphql filename="Example mutation: Add checkout billing address" showLineNumbers copy
+    mutation addCheckoutBillingAddress($addCheckoutBillingAddressInput: AddCheckoutBillingAddressInput!) {
+      checkout {
+        addCheckoutBillingAddress(input: $addCheckoutBillingAddressInput) {
+          checkout {
+            entityId
+          }
+        }
+      }
+    }   
+    ```
+
+    ```json filename="GraphQL variables" showLineNumbers copy
+    {
+      "addCheckoutBillingAddressInput": {
+        "checkoutEntityId": "812ece1a-da23-46eb-ab6b-c2ee210aae54",
+        "data": {
+          "address": {
+            "firstName": "Joe",
+            "lastName": "Blogs",
+            "email": "gqlbeta123@bigcommerce.com",
+            "company": "BC",
+            "address1": "123 Yar st",
+            "address2": "",
+            "city": "RWC",
+            "stateOrProvince": "CA",
+            "stateOrProvinceCode": "CA",
+            "countryCode": "US",
+            "postalCode": "94061",
+            "phone": "6501231212",
+            "shouldSaveAddress": false
+          }
+        }
+      }
+    }
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example mutation: Add checkout billing address" showLineNumbers copy
+    {
+      "data": {
+        "checkout": {
+          "addCheckoutBillingAddress": {
+            "checkout": {
+              "entityId": "bb916deb-ddd6-4586-b65b-b8385e0e7a9d"
```

Comment:
```suggestion
              "entityId": "812ece1a-da23-46eb-ab6b-c2ee210aae54"
```

I believe that the checkout entityId is just the checkout ID. If that's right, this should match the checkout entityId from the request.